scripts/checkpoint_converters/convert_mistral_7b_hf_to_nemo.py

```diff
@@ -329,6 +329,22 @@ def convert(args):
     model = model.to(dtype=dtype)
     model.cfg.use_cpu_initialization = False
 
+    if getattr(tokenizer, 'chat_template', None) is not None:
+        import hashlib
+
+        assert (
+            hashlib.md5(tokenizer.chat_template.encode('utf-8')).hexdigest() == "0b629f783db54e02509999196956ff40"
+        ), "Got unkown chat template"
+        from omegaconf import OmegaConf, open_dict
+
+        with open_dict(model.cfg):
+            model.cfg.tokenizer.chat_template = OmegaConf.create(
+                {
+                    'prefix': "{_bos_}",
+                    'roles': {'user': "[INST] {_content_} [/INST]", 'assistant': "{_content_}{_eos_}"},
```

Comment:
Perhaps it's better to set the roles as "User" and "Assistant" (capitalized) to match the NeMo convention. e.g. [here](https://github.com/NVIDIA/NeMo/blob/main/tests/collections/nlp/test_chat_sft_dataset.py#L64) and [here](https://github.com/NVIDIA/NeMo/blob/main/scripts/nlp_language_modeling/sft/preprocessing.py#L78)

Reply:
thanks @gleibovich-nvidia

Reply:
but keep in mind this is (currently) only relevant to the formatting of the input data to the eval script.